schema/requestedDate.json

```diff
@@ -0,0 +1,20 @@
+{
+    "id": "requestedDate.json",
+    "$schema": "https://json-schema.org/draft-04/schema#",
+    "description": "range of date validity",
+    "type": "object",
+    "example": "../examples/requestedDate.json",
+    "additionalProperties": false,
+    "required": ["requestedDateOfCollection","latestAcceptableDate"],
+    "properties": {
+        "requestedDateOfCollection": {
+            "type": "string",
+            "description": "preferred date and time of collection"
+        },
+        "latestAcceptableDate": {
+            "type": "string",
+            "description": "latest date and time collection must be completed"
+        }
```

Comment:
Let's use the common `date` type for this - there is a common.json file that contains this definition, here is an example of how it's referenced inside the citation:

```
"date": {
      "type": "array",
      "description": "Date referenced to the cited resource.",
      "translation": {
        "ISO 19115-2": ["CI_Citation > date > CI_Date"]
      },
      "items": {
        "$ref": "common.json#/definitions/dateObject"
      }
    },